msprime/simulations.py

```diff
@@ -1929,12 +1929,49 @@ def get_ll_representation(self):
 
 
 class BetaCoalescent(ParametricSimulationModel):
-    # TODO document.
+    """
+    A diploid Xi-coalescent with up to four simultaneous multiple mergers and
+    crossover recombination. The string ``"beta"`` can be used to refer to this
```

Comment:
I don't think we can use "beta" because this is a parametric model. The strings like "smc" are only available as a shortcut for non parametetric models.

Reply:
I picked this up from the `name = "beta"` line under the model class in simulations.py, which was already there before I started. I did wonder how that could work. I guess we should give the whole ``msprime.BetaCoalescent(alpha, truncation_point)`` string instead?

Reply:
We should just remove support for creating these classes from strings - it's not helpful I think. We can follow up in an issue, no need to solve the problem here.

Reply:
+1 for configuration classes over strings